### DISCUSSION TITLE
Any possibility to require approval for content changes using the current permissions structure?

### DISCUSSION THREAD

**Discussion by chase-replicated** [outline/outline#2084] [Ideas]
We intend to use outline to host our company handbook / wiki. It would be handy to have a git-like approval feature to ensure the involvement of multiple members or even specific members in any official change to company policy.

I'm interested to see anyone's opinion on whether this would be possible or anywhere to start. I'd be happy to eventually dig into this myself although I can't say I'm particularly skilled in Node. I'll be sure to followup on this thread if I have a chance to put something together in the future.

**Comment by tommoor** [comment]
Hi Chase,

There is an issue tracking this requested feature here – https://github.com/outline/outline/issues/1084 – as it stands today there isn't a way to achieve this however you can change a collection to have edit permissions for only a subset of users which would resolve it for many common cases.
